src/altinn-app-frontend/package.json

```diff
@@ -30,6 +30,7 @@
     "axios": "0.27.2",
     "dot-object": "2.1.4",
     "jsonpointer": "5.0.1",
+    "jss-rtl": "0.3.0",
```

Comment:
I'm not too sure about introduction of this dependency. It seems to be a relatively little used project (~80 stars on github, not updated in 3 years, ~50k weekly downloads on npm). We are also in general moving away from css-in-js.

Do you have examples of any specific rules that broke without this plugin? I think this is also valuable info for @Febakke and @mrosvik moving forward, that we probably need to also think about RTL languages when making our designs.

Reply:
It was the recommended way to handle rtl in mui v4: https://v4.mui.com/guides/right-to-left/#right-to-left

Reply:
The plugin will flip for example left padding to right padding in all mui components. It will basically flip everything mui related. And since it is alot of mui stuff here it was the easiest solution currently available.

Reply:
I'm just wondering if this is nescessary at all? We usually have equal padding on right and left sides, so there really shouldnt be much need to swap them. There might be edge cases here though, thats why I would also love some input from our UX group to see if they know more.

I did some quick checks with just enabling 

```css
direction: rtl;
text-align: right;
```
and from what I can tell it looks fine with just this? But I havent been able to really dig into this and look at everything either, so there may be things you have noticed, that doesnt work with just these settings.

Reply:
When removing this library the form looks like this. As you can see there are three problems. Right align text in table, Extra margin on right sides og RadioButtons(Will also apply for Checkboxes), Margin between NavigationButtons.
![image](https://user-images.githubusercontent.com/43168787/190596252-120e1f37-c251-480b-b2d3-178c748114e3.png)

Reply:
I have pushed a new commit where I remove the library and make the margin between Navigation and table cell right align text work without this library. The right margin on the Radiobutton isn't that important to fix imo. SO maybe we can live with the extra margin on the right side until these components are implemnted in the design system?